tests/test_code_coverage.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
-docker build --no-cache -t code_coverage:2.0 -f ./tests/code_coverage.Dockerfile .
-docker run --rm --security-opt seccomp=unconfined -it code_coverage:2.0
+docker build --no-cache -t code_coverage:1.0 -f ./tests/code_coverage.Dockerfile .
+docker run --rm --security-opt seccomp=unconfined -it code_coverage:1.0
```

Comment:
Looks like merge conflicts

```suggestion
docker build --no-cache -t code_coverage:2.0 -f ./tests/code_coverage.Dockerfile .
docker run --rm --security-opt seccomp=unconfined -it code_coverage:2.0
```

Reply:
I deliberated changed this back to version 1.0; I was having an issue on my machine with 2.0 - and we explicitly use `--rm` with docker run so the version should never increment.

Reply:
That not what `--rm` does 😕 

"remove" only cleans up the container not the image that is created.

What issue did you have? when creating an image you can give it any name. There's likely a bigger issue if you can not change the name.

Reply:
code_coverage is the name 2.0 is a tag. 2.0 implies there was a 1.0 tag that isn't published. I guess I want to know why we would skip 1.0 and jump to 2.0? If you really feel strongly about it I'll investigate further into the issue I had.

From `docker build --help`, `-t, --tag list                Name and optionally a tag in the 'name:tag' format`

Reply:
We will never publish this. It's just a local test image.
I changed the version because I changed the internal structure and broke the API on the image.

Let's remove version (and the confusion) seems like its worse to have it

```suggestion
docker build --no-cache -t code_coverage -f ./tests/code_coverage.Dockerfile .
docker run --rm --security-opt seccomp=unconfined -it code_coverage
```